.changelog/2687.feature.1.md

```diff
@@ -0,0 +1,25 @@
+`oasis-test-runner`: Add benchmarking utilities
+
+- `oasis-test-runner` now accepts `--metrics.address` and
+  `--metrics.push.interval` paramaters which are forwarded to `oasis-node`
+  workers.
+- `oasis-test-runner` now signals `up` metric to Prometheus when a test is
+  started and finished.
+- `--num_runs` parameter added which specifies how many times each test should
+  be run.
+- E2E scenario parameters can now be exposed and set via CLI by reimplementing
+  `scenario.Parameters()` and setting it with
+  `--params.<testname>.<param>=val1,val2,...`.
```

Comment:
Do you need to override parameters for each scenario? For example if you run all scenarios and they all share a specific parameter -- do you need to specify the same parameter multiple times? This would be annoying.

Reply:
Yeah, for now parameters are scenario-based. In the future I thought of having tree-like parameter sets, from the most global ones e.g. `--params.<param>`, `--params.<testname>.<param>`, `--params.<testname>.<subtestname>.<param>` etc.

Reply:
Recursive params are now implemented. Fixed.